From ee17cb1002d2b3b39a51e6cd5e26992036110090 Mon Sep 17 00:00:00 2001
From: Chris Crump <chriscrawford893@gmail.com>
Date: Thu, 8 Sep 2022 01:34:44 -0400
Subject: [PATCH] Fallback to the same certificate as platform

Using a separate certificate caused issues with seapp context, probably due to preparation for apex, but since apex isn't ready yet, let's fallback to platform

Change-Id: I4724fc49bbfc670c3900429671963b5e29c15a02
---
 android/app/Android.bp            | 7 +------
 android/app/tests/unit/Android.bp | 2 +-
 2 files changed, 2 insertions(+), 7 deletions(-)

diff --git a/android/app/Android.bp b/android/app/Android.bp
index ca46a38c..19192250 100644
--- a/android/app/Android.bp
+++ b/android/app/Android.bp
@@ -87,7 +87,7 @@ android_app {
         ":bluetooth-proto-enums-java-gen",
     ],
     aaptflags: [ "--custom-package", "com.android.bluetooth" ],
-    certificate: ":com.android.bluetooth.certificate",
+    certificate: "platform",
 
     jni_libs: ["libbluetooth_jni"],
     libs: [
@@ -163,8 +163,3 @@ genrule {
         + " --minApiLevel 33",
     out: ["com/android/bluetooth/BluetoothStatsLog.java"],
 }
-
-android_app_certificate {
-    name: "com.android.bluetooth.certificate",
-    certificate: "certs/com.android.bluetooth"
-}
diff --git a/android/app/tests/unit/Android.bp b/android/app/tests/unit/Android.bp
index ab9fe68b..2b82553e 100755
--- a/android/app/tests/unit/Android.bp
+++ b/android/app/tests/unit/Android.bp
@@ -6,7 +6,7 @@ android_test {
     name: "BluetoothInstrumentationTests",
 
     // We only want this apk build for tests.
-    certificate: ":com.android.bluetooth.certificate",
+    certificate: "platform",
 
     libs: [
         "javax.obex",
-- 
2.34.1

